rs/cli/src/auth.rs

```diff
@@ -87,8 +87,29 @@ impl Neuron {
         Self::from_opts_and_req_inner(auth_opts, requirement, network, neuron_id).await
     }
 
-    pub async fn dry_run_fake_neuron(network: &Network) -> anyhow::Result<Self> {
-        Self::from_opts_and_req(AuthOpts::none(), AuthRequirement::Anonymous, network, None).await
+    fn ensure_fake_pem(name: &str) -> anyhow::Result<PathBuf> {
```

Comment:
```suggestion
    fn new_with_fake_pem(name: &str) -> anyhow::Result<PathBuf> {
```

Reply:
But it just ensures that a file on that path exists. It doesn't return `Self`?

Reply:
But it just ensures that a path exists. It doesn't return `Self`?

Reply:
ah right, my bad, sorry